### PR TITLE
New curators and 3 curators promoted to manager

### PR DESCRIPTION
**Pull request by leahcimto**
Hi! I updated curators.json and managers.json with new curators and I moved the curators that were recently promoted to managers.json.

### CODE REVIEW

File: Forum_Helpers/managers.json [hunk 1]
```diff
@@ -277,5 +277,27 @@
...
+	
+    {
+        "name": "xXRedTheCoderXx",
+        "id": 25035668,
+        "bio": "Hiya! My name is Red, and I like to change my profile picture to something wacky everyday. Feel free to say hi, or ask about something on my profile. :) I'm friendly (usually :P) and I gladly help users here on Scratch. Tie your shoes, and Scratch On!",
+        "helped_code": false
+    },
+	
+    {
+        "name": "fdreerf",
```
Comment (Accio1) [inline Forum_Helpers/managers.json:290]: Fdreerf should be before xXRedTheCoderXx

File: Forum_Helpers/managers.json [hunk 2]
```diff
@@ -277,5 +277,27 @@
...
+        "bio": "why does this site use comic sans",
+        "helped_code": false
+    },
+	
+    {
+        "name": "Super_Scratch_Bros20",
+        "id": 35057879,
+        "bio": "No Biography Provided.",
+        "helped_code": false
     }
```
Comment (Accio1) [inline Forum_Helpers/managers.json:301]: Please add Paddle2See

**Review by Accio1 — CHANGES_REQUESTED** [review]
@leahcimto Curators List looks good. Please make the indicated changes to the Managers list, then I will review this again and hopefully merge it soon!

**Comment by leahcimto** [comment]
Thanks, will add those changes today.

**Comment by leahcimto** [comment]
I made the changes, let me know if anything else should be changed.

**Comment by Accio1** [comment]
I'm at school right now so I can't review it. I'm labelling it as unreviewed so that I remember to review it tonight.

**Review by Accio1 — APPROVED** [review]
Thanks for doing this! Looking over it briefly, it looks good to merge!

**Comment by Accio1** [comment]
This is partially my fault for not checking, but in the future, please make sure to remove the comma after the final item in the list, as that causes an error when the site tries to load it.

**Pull request merged**